Source/BugsnagEvent.m

```diff
@@ -26,6 +26,12 @@
 #import "BugsnagKeys.h"
 #import "BugsnagClient.h"
 
+@interface BugsnagAppWithState ()
++ (BugsnagAppWithState *)appWithDictionary:(NSDictionary *)event config:(BugsnagConfiguration *)config;
+- (NSDictionary *)toDict;
++ (BugsnagAppWithState *)appWithOomData:(NSDictionary *)event;
+@end
+
```

Comment:
It would be good if the class extensions could be collected together and `// MARK:-`ed as a group to help delineate dependencies from the main body of the class.

Reply:
Added a few comments and rearranged accordingly.